src/ingest_validation_tools/enums.py

```diff
@@ -1,12 +1,93 @@
 from enum import Enum, unique
```

Comment:
moved deprecated assay type enum to end of file, removed TODO, fixed typing - no code changes